pkg/utils/errors.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package utils
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+)
+
+const noLimitNumberOfMatches = -1
+
+var (
+	filters = []string{
+		"failing health check",
+		"DEBUG",
+		"INFO",
+	}
+	// It was initially suggested to also look for all warn messages,
+	// however this results in too much stuff printed to the screen.
+	// For documentation reasons though leaving the according regex here
+	// so it can easily be re-enabled if wished.
+	// errorRegEx := regexp.MustCompile(`(?im)(^.*error.*|.*warn.*)`)
+	errorRegEx = regexp.MustCompile(`(?im)(^.*error.*)`)
+)
+
+// FindErrorLogs is a utility function, we will NOT do error handling,
+// as this is supposed to be called during error handling itself
+// we don't want to make it even more complex
+func FindErrorLogs(rootDirs ...string) {
+	alreadyNotified := false
+	foundErrors := false
+
+	for _, rootDir := range rootDirs {
+		if _, err := os.Stat(rootDir); err != nil {
+			return
+		}
+
+		_ = filepath.WalkDir(rootDir, func(path string, d fs.DirEntry, err error) error {
```

Comment:
ahhh, you took a different approach than I was expecting. I thought we should just print root/node1/logs/XYZ.log. I think this code will replicate the same errors from every node, right?

Maybe default behavior should be we only look at the custom chain log from one node, then if you pass the `--debug` flag, we print everything here?

Reply:
This is reasonable. But, I have seen situations where errors appeared just in one log file, although probably less frequent. We could miss errors this way.

Reply:
Tried different approach: store found logs, and compare new ones with already found, and skip if already found. For already found, the criteria is to strip away the timestamp and compare the whole remaining string.